lib/index.js

```diff
@@ -343,6 +351,9 @@ var getSelfLaunchURL = function (opts) {
     if (appId) {
         url += "&appId=".concat(appId);
     }
+    if (selfLaunchState) {
+        url += "&selfLaunchState=".concat(encodeURIComponent(JSON.stringify(selfLaunchState)));
+    }
```

Comment:
Does the client pass selfLaunchState data to the server via URL param? Would it be safer to pass that info in the body?